### ISSUE TITLE
[package_info_plus_web] Not working on 0.6.2

### ISSUE THREAD

**Issue by jlubeck** [fluttercommunity/plus_plugins#72]
## System info

Issue occurs on: web
Plugin name: package_info_plus
Plugin version: 0.6.2
Flutter doctor output:

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel dev, 1.25.0-8.0.pre, on macOS 11.0.1 20B50 darwin-x64,
    locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 12.2)
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1)
[✓] VS Code (version 1.52.0)
[✓] Connected device (2 available)

• No issues found!
```

## Steps to Reproduce

I have this widget that I use to show the version wherever I want. 

```dart
import 'package:flutter/material.dart';
import 'package:package_info_plus/package_info_plus.dart';

class BuildNumberWidget extends StatefulWidget {
  @override
  _BuildNumberWidgetState createState() => _BuildNumberWidgetState();

  final TextStyle style;
  final String preText;
  final String postText;
  BuildNumberWidget({this.style, this.preText = '', this.postText = ''});
}

class _BuildNumberWidgetState extends State<BuildNumberWidget> {
  PackageInfo _packageInfo = PackageInfo(
    appName: 'Unknown',
    packageName: 'Unknown',
    version: 'Unknown',
    buildNumber: 'Unknown',
  );

  @override
  void initState() {
    super.initState();
    _initPackageInfo();
  }

  Future<void> _initPackageInfo() async {
    final PackageInfo info = await PackageInfo.fromPlatform();
    setState(() {
      _packageInfo = info;
    });
  }

  @override
  Widget build(BuildContext context) {
    return Text(
      '${widget.preText}${_packageInfo.version} (${_packageInfo.buildNumber})${widget.postText}',
      style: widget.style,
    );
  }
}

```

On 0.5.0 works perfectly. On 0.6.2 returns null for every property

**Comment by jpnurmi** [comment]
Thanks! Fixed in 0.6.3

**Comment by jlubeck** [comment]
That was fast!
